core/src/main/java/org/zenoss/zep/index/impl/EventIndexerImpl.java

```diff
@@ -305,12 +310,17 @@ public void handleDeleted(String uuid) throws Exception {
 
             @Override
             public void handleComplete() throws Exception {
+                indexDao.commit();
```

Comment:
This is the real meat of the change - committing to the index before invoking plugins.